static/app/components/events/featureFlags/featureFlagDrawer.tsx

```diff
@@ -102,6 +105,10 @@ export function FeatureFlagDrawer({
         }}
         onChange={selection => {
           setSortMethod(selection.value);
+          trackAnalytics('flags.sort-flags', {
+            organization,
+            sortMethod: selection.value,
+          });
```

Comment:
is it important to know this is coming from drawer vs on issue details?

Reply:
i don't know if there's a big distinction to me, wdyt?